### PR TITLE
Added SEPP easyblock

### PR DESCRIPTION
**Pull request by Darkless012**

### CODE REVIEW

File: easybuild/easyblocks/s/sepp.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+        # Configure sepp
+        run_cmd("python setup.py config -c")
+
+    def install_step(self, *args, **kwargs):
+        super(EB_SEPP, self).install_step(*args, **kwargs)
+
+        # Python version
+        pyver = ".".join(get_software_version('Python').split(".")[:-1])
+        python_site_packages_dir = os.path.join(self.installdir, "lib", "python%s" % pyver, "site-packages")
```
Comment (boegel) [inline easybuild/easyblocks/s/sepp.py]: You can just use `self.pylibdir` (which is set by `PythonPackage.configure_step`)

File: easybuild/easyblocks/s/sepp.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        # Old (wrong) SEPP paths
+        sepp_wrong_config_dir = os.path.join(self.builddir, self.name.lower()+"-"+self.version, '.sepp')
```
Comment (boegel) [inline easybuild/easyblocks/s/sepp.py]: spaces around `+` please (I'm surprised Hound doesn't complain about that)

File: easybuild/easyblocks/s/sepp.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+        # New (correct) SEPP paths
+        sepp_correct_home_path_file = os.path.join(python_site_packages_dir, "home.path")
```
Comment (boegel) [inline easybuild/easyblocks/s/sepp.py]: please try to use `'` consistently for string values that don't have spaces (why `"` here and `'` below)

File: easybuild/easyblocks/s/sepp.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        sepp_correct_config_dir = os.path.join(python_site_packages_dir, '.sepp')
+        sepp_correct_config_file = os.path.join(sepp_correct_config_dir, 'main.config')
+
+        # Create correct home.path file
+        self.log.debug("Creating home.path file for SEPP at %s", sepp_correct_home_path_file)
```
Comment (boegel) [inline easybuild/easyblocks/s/sepp.py]: `log.info` makes more sense here I think

**Comment by Darkless012** [comment]
Wrong develop, closing. Will reopen with correct one.

File: easybuild/easyblocks/s/sepp.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        write_file(sepp_correct_home_path_file, sepp_correct_config_dir)
+
+        # Copy .sepp folder with configurations and bundled stuff from builddir to installdir
+        copy_dir(sepp_wrong_config_dir, sepp_correct_config_dir)
+
+        # Replace wrong paths in SEPP configfile with correct ones
+        run_cmd("sed -i 's#%s#%s#g' %s" % (sepp_wrong_config_dir, sepp_correct_config_dir, sepp_correct_config_file))
```
Comment (boegel) [inline easybuild/easyblocks/s/sepp.py]: we have the `apply_regex_substitutions` function for runtime patching, please don't rely on `sed` for this

**Comment by boegel** [comment]
replaced with #1998

**Pull request closed without merging**